util/reformat-yaml/reformat.go

```diff
@@ -0,0 +1,120 @@
+package main
+
```

Comment:
Can you add a `README.md` file within the reformat-yaml directory, which gives a brief explanation of what this script does (i.e. reformats specs to work with v2 syntax) and how to use it?